WebDriverAgentLib/Commands/FBFindElementCommands.m

```diff
@@ -103,6 +103,10 @@ + (NSArray *)routes
 
 + (XCUIElement *)elementUsing:(NSString *)usingText withValue:(NSString *)value under:(XCUIElement *)element
 {
+  if ([usingText isEqualToString:@"xpath"]) {
```

Comment:
Ok, I see. You could add BOOL parameter to `+ (NSArray *)elementsUsing:(NSString *)usingText withValue:(NSString *)value under:(XCUIElement *)element` something like `canQuitOnFirstMatch` and past it to `fb_descendantsMatchingXPathQuery`. I don't like copy pasting same code around